src/Pheno-Tests-Core.package/PHDummyToolBuilder.class/class/isActive..st

```diff
@@ -0,0 +1,5 @@
+testing
+isActive: aBoolean
+
+	IsActive := aBoolean.
+	UIManager default builderClass: nil.
```

Comment:
hm does this work for you to keep the Pheno ToolBuilder active? I just had a quick read-through the nil handling and came to believe that it would always fallback to the Morphic toolbuilder after running the tests.

My guess would have been that we need to remember the previous choice and restore it as is

Reply:
Provided that the pheno preference is enabled, `UIManager default builderClass: nil; builderClass` returns a PhenoToolBuilder, at least for me ... This is because PhenoToolBuilder is a subclass of MorphicToolBuilder. What issue are you facing?

Reply:
My hypothesis was that if I have two subclasses of MorphicToolBuilder and set mine explicitly via `UIManager default builderClass: MySecondMorphicToolBuilder`, resetting it to nil after the tests might end up picking the PhenoToolBuilder as it may come last in the list of subclasses. But I feel like I also just may have misunderstood `findDefault` of UIManager.

Reply:
I don't think it would be a good idea to manipulate the builderClass manually. For example, there are some external users of #findDefault ...

Reply:
Alright, looking great then! thank you :)